canvas_modules/common-canvas/src/common-properties/panels/tearsheet/tearsheet.scss

```diff
@@ -0,0 +1,21 @@
+.properties-tearsheet-panel {
+	.bx--modal-header {
+		position: absolute;
+		right: 0;
+	}
+	.bx--modal-content {
+		padding: 0;
+		margin: 0;
+		.properties-tearsheet-header {
+			padding: $spacing-06 $spacing-07;
+			border-bottom: 1px solid $gray-20;
```

Comment:
This doesn't look right.  Color tokens come from here https://v10.carbondesignsystem.com/guidelines/color/usage/